htlcswitch/link.go

```diff
@@ -1091,15 +1091,20 @@ func (l *channelLink) htlcManager() {
 			}
 
 			// We'll check to see if we should update the fee rate
-			// based on our current set fee rate. We'll cap the new
-			// fee rate to our max fee allocation.
+			// based on our current set fee rate, the target
+			// network fee rate and the minimum relay fee.
+			// We'll cap the new fee rate to our max fee allocation.
 			commitFee := l.channel.CommitFeeRate()
 			maxFee := l.channel.MaxFeeRate(
 				l.cfg.MaxFeeAllocation,
 				l.cfg.MaxAnchorsCommitFeeRate,
 			)
+			relayFee := l.cfg.FeeEstimator.RelayFeePerKW()
 			newCommitFee := chainfee.SatPerKWeight(
-				math.Min(float64(netFee), float64(maxFee)),
+				math.Max(float64(netFee), float64(relayFee)),
+			)
+			newCommitFee = chainfee.SatPerKWeight(
+				math.Min(float64(newCommitFee), float64(maxFee)),
```

Comment:
naming `math.Max(float64(netFee), float64(relayFee))` `newCommitFee` is quite confusing, `minCommitFee`?